README.md

```diff
@@ -39,7 +39,7 @@ scripts/kconfig/merge_config.sh .config <PATH>/kconfig/base-defconfig <PATH>/kco
 ````
 - SOF-DEFAULT: SOF support with I2S, HDA and SDW support with machine drivers
 ````bash
-scripts/kconfig/merge_config.sh .config <PATH>/kconfig/base-defconfig <PATH>/kconfig/sof-defconfig <PATH>/kconfig/sof-mach-driver-defconfig <PATH>/kconfig/hdaudio-codecs-defconfig <PATH>/kconfig/soundwire-defconfig
+scripts/kconfig/merge_config.sh .config <PATH>/kconfig/base-defconfig <PATH>/kconfig/sof-defconfig <PATH>/kconfig/sof-dev-defconfig <PATH>/kconfig/hdaudio-codecs-defconfig <PATH>/kconfig/soundwire-defconfig
```

Comment:
actually this fix is also wrong, the machine drivers need to be there but where renamed as mach-driver-defconfig
Probably the best is to refer to the latest sof-default.sh, instead of maintaining the documentation by hand

Reply:
@plbossart ok, we can drop that patch, it just cost me a few hours to figure out why my configuration was broken...